src/Classes/ServiceAPI/MyRadio_Timeslot.php

```diff
@@ -216,6 +222,35 @@ public function getEndTime()
         return $this->getStartTime() + $duration;
     }
 
+    /**
+     * Returns the events the timeslot is a part of
+     * 
+     * @return MyRadio_Event[]
+     */
+    public function getEvents(){
+        return MyRadio_Event::resultSetToObjArray($this->events);
+    }
+
+    /**
+     * Can add this timeslot to an event
+     * 
+     * @param $event_id
+     * 
+     * @return MyRadio_Event
+     * 
+     * @throws MyRadioException
+     */
+
+     public function addEvent($event_id){
```

Comment:
Not sure how I feel about having the same-ish method in both Event and Timeslot… but feel free to overrule me

Reply:
can't hurt

Reply:
it can actually